central/processlisteningonport/datastore/datastore.go

```diff
@@ -0,0 +1,43 @@
+package datastore
+
+import (
+	"context"
+	"fmt"
+
+	processIndicatorStore "github.com/stackrox/rox/central/processindicator/datastore"
+	"github.com/stackrox/rox/central/processlisteningonport/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// GetOptions specifies how to get information from the database, simply
+// filtering by the namespace, or both namespace and deployment
+type GetOptions struct {
+	DeploymentID *string
+	Namespace    *string
+}
+
+func (opts *GetOptions) String() string {
+	return fmt.Sprintf("GetOptions{Namespace: %s, DeploymentID: %s}",
+		*opts.Namespace, *opts.DeploymentID)
+}
+
+// DataStore interface for ProcessListeningOnPort object interaction with the database
+//go:generate mockgen-wrapper
+type DataStore interface {
+	AddProcessListeningOnPort(context.Context, ...*storage.ProcessListeningOnPort) error
+	GetProcessListeningOnPort(
+		ctx context.Context,
+		opts GetOptions,
+	) (map[string][]*storage.ProcessListeningOnPort, error)
+}
+
+// New creates a data store object to access the database. Since some
```

Comment:
subtle distinction.  This includes the PLOP store but the process indicator datastore.  comment states using the process indicator store directly.

Reply:
Good point. What is the common practice to distinguish both, are "store" and "datastore" different enough?

Reply:
yes generally using store vs datastore in the name is sufficient.  From time to time I've even seen datastore as DS.  but I think I prefer it spelled out.